parts/windows/kuberneteswindowsfunctions.ps1

```diff
@@ -139,11 +141,15 @@ function Retry-Command {
         $RetryDelaySeconds
     )
 
-    for ($i = 0; $i -lt $Retries; $i++) {
+    for ($i = 0; ; ) {
         try {
             return & $Command @Args
         }
         catch {
+            $i++

```

Comment:
curious how to write the unit test for powershell.

Reply:
Good point. We have not written unit tests for cse. I have validated these code in the node directly. Later we will vendor this change to RP and then validate it again.